build.rs

```diff
@@ -450,6 +450,10 @@ mod c {
                 ("__floatunsitf", "floatunsitf.c"),
                 ("__fe_getround", "fp_mode.c"),
             ]);
+
+            if target_env == "musl" {
+                sources.extend(&[("__divtf3", "divtf3.c"), ("__trunctfdf2", "trunctfdf2.c")]);
```

Comment:
Add these to the list above, there's no need to restrict these to musl only.

Reply:
Done